src/System.Net.Sockets/tests/FunctionalTests/NetworkStreamTest.cs

```diff
@@ -623,6 +623,7 @@ public void Ctor_NotConnected_ThrowsIOException()
         [InlineData(4096)]
         [InlineData(4095)]
         [InlineData(1024*1024)]
+        [ActiveIssue(20135, TargetFrameworkMonikers.Uap)]
```

Comment:
I looked at the issue, but there's not much detail in there.  Is the relevant work known / being tracked somewhere?